### ISSUE TITLE
License of the project?

### ISSUE THREAD

**Issue by forestrf** [pipe/sctp4j#1]
Is the license of this project apache v2, or is it GNU GPL v2?
Some files (nearly all of them) are apache v2, but some test cases are GNU GPL v2

Thank you

**Comment by steely-glint** [comment]
Oh, let me check. 
The intention is that it should be all apache v2.

**Comment by steely-glint** [comment]
I've had a quick look, and I can't see any GPL licenses in the current release can you point me at any please ?

**Comment by forestrf** [comment]
Yes. I had to search for GNU as I didn't remember which files, and this is one of them that is GPL v2 https://github.com/pipe/sctp4j/blob/ae7de1c337d2814bf4d53070911ec4a5a28e964f/test/main/java/com/ipseorama/sctp/MockAssociation.java

**Comment by steely-glint** [comment]
Ha. That should have been removed long ago. Thanks.
